lib/msal-browser/docs/initialization.md

```diff
@@ -99,7 +99,7 @@ The popup APIs use ES6 Promises that resolve when the authentication flow in the
 - `loginRedirect`
 - `acquireTokenRedirect`
 
-The redirect APIs are asynchronous (i.e. return a promise) `void` functions which redirect the browser window after caching some basic info. If you choose to use the redirect APIs, be aware that you MUST call `handleRedirectPromise()` to correctly handle the API. You can use the following function to perform an action when this token exchange is completed:
+The redirect APIs are asynchronous (i.e. return a promise) `void` functions which redirect the browser window after caching some basic info. If you choose to use the redirect APIs, be aware that **you MUST call `handleRedirectPromise()` to correctly handle the API**. You can use the following function to perform an action when this token exchange is completed:
```

Comment:
Prithvi and I were thinking about moving this section out of the initialization docs because it's confusing for angular/react users. For now can we add a note that if you're using msal-angular or msal-react this is done differently and include docs to the respective libraries?

Reply:
Sounds good. Added.